Assets/Scripts/UI/LobbyCodeInputTextField.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+namespace UI
+{
+    public class LobbyCodeInputTextField : MonoBehaviour, ITextInputFieldReader
+    {
+        public JoinLobbyButton _joinLobbyButton; 
```

Comment:
We are creating some unnecessary dependencies here which we are going to have to make sure we're keeping track of in the inspector, since that is how we are assigning them. The JoinLobbyButton script is certainly unnecessary, since it does not have unique functionality, it only calls a method from somewhere else. In the case of this script, LobbyCodeInputTextField, I am slightly uncertain as to the purpose of using this interfasce; I'm curious why we are not handling this functionality somewhere more easily accessible and adding a method directly to our InputField's OnEndEdit event, though this might be more transparent when viewing the entire codebase.